twake/backend/node/src/services/workspaces/web/controllers/workspace-invite-tokens.ts

```diff
@@ -0,0 +1,165 @@
+import { CrudController } from "../../../../core/platform/services/webserver/types";
+import {
+  ResourceCreateResponse,
+  ResourceDeleteResponse,
+  ResourceGetResponse,
+  ResourceListResponse,
+} from "../../../../utils/types";
+import WorkspaceServicesAPI from "../../api";
+import {
+  WorkspaceInviteTokenDeleteRequest,
+  WorkspaceInviteTokenGetRequest,
+  WorkspaceInviteTokenObject,
+  WorkspaceJoinByTokenRequest,
+  WorkspaceJoinByTokenResponse,
+} from "../types";
+import { FastifyInstance, FastifyReply, FastifyRequest } from "fastify";
+
+import { WorkspaceInviteTokensExecutionContext } from "../../types";
+import { CrudExeption } from "../../../../core/platform/framework/api/crud-service";
+import WorkspaceInviteTokens from "../../entities/workspace_invite_tokens";
+import Workspace from "../../entities/workspace";
+import { pick } from "lodash";
+
+export class WorkspaceInviteTokensCrudController
+  implements
+    CrudController<
+      ResourceGetResponse<WorkspaceInviteTokenObject>,
+      ResourceCreateResponse<WorkspaceInviteTokenObject>,
+      ResourceListResponse<WorkspaceInviteTokenObject>,
+      ResourceDeleteResponse
+    >
+{
+  constructor(protected services: WorkspaceServicesAPI) {
+    this.services = services;
+  }
+
+  async get(
+    request: FastifyRequest<{ Params: WorkspaceInviteTokenGetRequest }>,
+    reply: FastifyReply,
+  ): Promise<ResourceGetResponse<WorkspaceInviteTokenObject>> {
+    const context = getExecutionContext(request);
+
+    const res = await this.services.workspaces.getInviteToken(
+      context.company_id,
+      context.workspace_id,
+    );
+
+    if (!res) {
+      throw CrudExeption.notFound("Invite token not found");
+    }
+
+    return {
+      resource: { token: res.token },
```

Comment:
Can we make it more close to the v2 API? I mean return a list instead of a single object?